kustomize/rbac/role.yaml

```diff
@@ -4,6 +4,18 @@ kind: ClusterRole
 metadata:
   name: kwok-controller
 rules:
+- apiGroups:
```

Comment:
Add into 
https://github.com/kubernetes-sigs/kwok/blob/3df63f73098694fd0bf449353bce2669a0d7da01/pkg/apis/v1alpha1/doc.go#L21-L25

and run `./hack/update-controllergen.sh`

Reply:
Updated